packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -288,4 +340,43 @@ function AnimalInventory({
   );
 }
 
+export function ExpandableAnimalInventory(props: ExpandableAnimalInventoryProps) {
+  const { t } = useTranslation();
+  const { inventory } = useAnimalInventory();
+  const animalCount = props.preSelectedIds.reduce((acc, cur) => {
+    return acc + (inventory.find((animalOrBatch) => animalOrBatch.id === cur)?.count || 0);
+  }, 0);
```

Comment:
This is totally okay but I think I personally would find this a bit more readable

```js
const animalCount = inventory.find((animalOrBatch) => props.preSelectedIds.includes(animalOrBatch.id)).length;
```

Reply:
Its got to be `count` not `length` because of batches so I don't think I can do this.

Reply:
Ah!! I totally missed the count part, makes sense